src/components/ServiceSearch.jsx

```diff
@@ -39,31 +83,39 @@ const ServiceSearch = () => {
   return (
     <>
       {redirect && <Redirect to="/services/search" />}
-      <Box data-cy="self-care-search-bar" style={styles.searchBar}>
-        <FormControl>
-          <OutlinedInput
-            data-cy="search-query"
-            onChange={(e) => setSearchQuery(e.target.value)}
-            color="secondary"
-            value={searchQuery}
-            placeholder={"Search for a community service..."}
-            aria-describedby="Search for self care services"
-            style={styles.queryInput}
-          />
-          <FormHelperText style={styles.helperText}>
-            Try searching for "befriending", "chess" or "sports".
-          </FormHelperText>
-        </FormControl>
-        <Button
-          data-cy="search-submit"
-          onClick={() => performSearch()}
-          variant="contained"
-          color="secondary"
-          style={styles.searchButton}
-          disableElevation
+      <Box style={styles.fullWidth}>
+        <Box
+          data-cy="self-care-search-bar"
+          className={classes.searchBar}
+          style={styles.searchBar}
         >
-          <SearchIcon />
-        </Button>
+          <FormControl>
+            <OutlinedInput
+              data-cy="search-query"
+              onChange={(e) => setSearchQuery(e.target.value)}
+              color="secondary"
+              value={searchQuery}
+              placeholder={"Search for a community service..."}
+              aria-describedby="Search for self care services"
+              className={classes.queryInput}
+              style={styles.queryInput}
```

Comment:
same

Reply:
named it the same for ease of reading i thought but if you have a good name write it under here